examples/cfd/07_cavity_flow.ipynb

```diff
@@ -0,0 +1,700 @@
+{
```

Comment:
<p>All data should already be zero by default</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Thanks, I've removed assigning the variables to zero.